addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -366,3 +366,13 @@ $pagination-padding-x-lg: $pagination-padding-x !default;
 
 $pagination-border-radius-lg: $pagination-border-radius !default;
 $pagination-border-radius-sm: $pagination-border-radius !default;
+
+// Offcanvas
+$offcanvas-bg-color: $body-bg !default;
+
+// Button close
+$btn-close-color: $body-color !default;
+
+// List group
+$list-group-bg: $body-bg;
```

Comment:
Ok(-ish) since by default BS [uses white](https://github.com/twbs/bootstrap/blob/1a6fdfae6be09b09eaced8f0e442ca6f7680a61e/scss/_variables.scss#L1480).... but let's take in account that  `$body-bg` is editable by the user thus some other list-group properties defined by BS variables may fail.

You adapted  `$list-group-color` but there is much more like border, disabled states etc... 
https://github.com/twbs/bootstrap/blob/1a6fdfae6be09b09eaced8f0e442ca6f7680a61e/scss/_variables.scss#L1478

Reply:
I adapted the other variables I thought had to be adapted